zuul.d/projects.yaml

```diff
@@ -21,6 +21,12 @@
             files:
               - ^user_data_gpu/.*
 
+- project:
+    name: theopenlab/volcano
+    check:
+      jobs:
+        - tensorflow-benchmarks-volcano/kubeflow-k8s-v1.14.0
```

Comment:
it will run tensorflow benchmarks on volcano+k8s or tensorflow benchmarks on kubefow+k8s. do you think use `-` better ? if so will change it.

Reply:
Yes, please use `-`, slash `/` means directory in OpenLab context, for example: `playbooks/spark-integration-test-minikube-k8s`. So at the first time I see the name, I try to search directory `tensorflow-benchmarks-volcano` in the doc tree :)

Reply:
Or can we define two jobs for these two situation?

Reply:
@kiwik OK, will change @h00130372 two jobs per need 5 nodes, it will consume 10 nodes once, and this job for comparing the performance of volcano and kubeflow, we'd better keeper the env condition consistent.